python3 bindings fail to build if imgui disabled

#### System Information
- Ogre Version: 1.12.4 
- Operating System / Platform: Ubuntu 18.04 
- RenderSystem: :grey_question:

#### Detailled description
I'm trying to update the ogre package released for Debian (and Ubuntu) to the current 1.12.4 and to enable some language bindings as well. As imgui is not included in the release tarball I'm currently disabling it (I read about #1332 ) by setting `-DOGRE_BUILD_COMPONENT_OVERLAY_IMGUI:BOOL=FALSE`

However it then fails to find the imgui include later on if I enable python bindings generation:



#### log
```
[ 97%] Building CXX object Components/Python/CMakeFiles/_Overlay.dir/OgreOverlayPYTHON_wrap.cxx.o
cd /home/simon/proggn/ogre/salsa_ogre/ogre/obj-x86_64-linux-gnu/Components/Python && /usr/bin/c++  -D_Overlay_EXPORTS -I/usr/include/python3.6m -I/home/simon/proggn/ogre/salsa_ogre/ogre/obj-x86_64-linux-gnu/include -I/home/simon/proggn/ogre/salsa_ogre/ogre/OgreMain/include -I/home/simon/proggn/ogre/salsa_ogre/ogre/Components/Overlay/include  -Wall -Winit-self -Wcast-qual -Wwrite-strings -Wextra -Wundef -Wmissing-declarations -Wno-unused-parameter -Wshadow -Wno-missing-field-initializers -Wno-long-long -g -O2 -fdebug-prefix-map=/home/simon/proggn/ogre/salsa_ogre/ogre=. -fstack-protector-strong -Wformat -Werror=format-security -pipe -Wall -Wdate-time -D_FORTIFY_SOURCE=2 -Wdate-time -D_FORTIFY_SOURCE=2 -msse -fPIC -fvisibility=hidden -fvisibility-inlines-hidden   -Wno-cast-qual -Wno-shadow -Wno-maybe-uninitialized -Wno-unused-label -Wno-unused-but-set-variable -Wno-missing-declarations -std=c++11 -o CMakeFiles/_Overlay.dir/OgreOverlayPYTHON_wrap.cxx.o -c /home/simon/proggn/ogre/salsa_ogre/ogre/obj-x86_64-linux-gnu/Components/Python/OgreOverlayPYTHON_wrap.cxx
/home/simon/proggn/ogre/salsa_ogre/ogre/obj-x86_64-linux-gnu/Components/Python/OgreOverlayPYTHON_wrap.cxx:4673:10: fatal error: imgui.h: Datei oder Verzeichnis nicht gefunden
 #include "imgui.h"
          ^~~~~~~~~
compilation terminated.
```
#### fix

seems to be a missing conditional in OgreOverlay.i if I change lines 23& 24 to 

```
#ifdef HAVE_IMGUI
#include "imgui.h"
#include "OgreImGuiOverlay.h"
#endif
```
it compiles successfully

--- a/Components/Overlay/include/OgreOverlay.i
+++ b/Components/Overlay/include/OgreOverlay.i
@@ -20,9 +20,13 @@
 #include "OgrePanelOverlayElement.h"
 #include "OgreTextAreaOverlayElement.h"
 
+%}
+#ifdef HAVE_IMGUI
+%{
 #include "imgui.h"
 #include "OgreImGuiOverlay.h"
 %}
+#endif
 
 %include std_string.i
 %include exception.i 

